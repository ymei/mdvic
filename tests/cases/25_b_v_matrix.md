Inline: $\begin{bmatrix} a & b \\ c & d \end{bmatrix}$ and $\begin{vmatrix} 1 & 2 \\ 3 & 4 \end{vmatrix}$.

Display:
$$\begin{bmatrix} 11 & 2 \\ 3 & 444 \end{bmatrix}$$

