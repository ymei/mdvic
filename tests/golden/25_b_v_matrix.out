Inline: [  a   b ;  c   d  ] and |  1   
2 ;  3   4  |.

Display: 
[  11   2    ]
[  3    444  ]
